src/modules/chatExtras/assets/lightDesignChatHistory.ts

```diff
@@ -0,0 +1,18 @@
+export default (redesignActive: boolean, modalName: string) => {
+    let startPhrase = redesignActive ? `#modals-container [data-modal="${modalName}"] + .vm--modal .redesign-wrapper ` : '';
+    document.querySelectorAll<HTMLSpanElement>(`${startPhrase}div.well span.pull-right`).forEach((el: HTMLSpanElement) => {
+        el.classList.remove('pull-right');
+        el.textContent = '[' + el.textContent + ']';
+    });
+    document.querySelectorAll<HTMLDivElement>(`${startPhrase}div.well`).forEach((el: HTMLDivElement) => {
+        el.classList.remove('well');
+        el.classList.add('chat-element-container');
+    });
+    document.querySelectorAll<HTMLParagraphElement>(`${startPhrase}div.chat-element-container p`).forEach((el: HTMLParagraphElement) => {
+        if(!el.classList.contains('label')) {
```

Comment:
In which case is there a  paragraph with the class label?

Reply:
there is such a element, when the redesign is inactive. It shows the info `Nachrichten werden nach 3 tagen gelöscht.`